backend-docker/server/apiserver.go

```diff
@@ -325,13 +336,109 @@ func (s *Server) buildVolumeHandler(w http.ResponseWriter, r *http.Request) {
 			Response{w}.ServerError("while dumping file part ", err)
 			return
 		}
+	}*/
+
+	// STEP 2: create a bash script that downloads those files
+	dScriptPath := filepath.Join(buildDir, "downloader.sh")
+	dScriptFile, err := os.Create(dScriptPath)
+	if err != nil {
+		Response{w}.ServerError("create script file:", err)
+		return
+	}
+	log.Println("Script was created")
+	_, err = dScriptFile.WriteString(strings.Join(pidList, "\n"))
+	if err != nil {
+		Response{w}.ServerError("write data into the script file", err)
+		return
 	}
+	dScriptFile.Sync()
+	log.Println("Wrote file list")
+
+	err = dScriptFile.Chmod(0777)
+	if err != nil {
+		Response{w}.ServerError("make downloading script executable:", err)
+		return
+	}
+	log.Println("Changed permissions")
+
+
+
 
+	// STEP 3: create an image that includes the script
+	var dockerFileContent []string
+	dockerFileContent = append(dockerFileContent, "FROM alpine:latest")
```

Comment:
put the entire script in a single string, see http://stackoverflow.com/questions/7933460/how-do-you-write-multiline-strings-in-go